src/redux/books/books.js

```diff
@@ -0,0 +1,24 @@
+export const addBook = (item) => ({
+  type: 'ADD',
+  item,
+});
+
+export const removeBook = (index) => ({
+  type: 'REMOVE',
+  index,
+});
+
+const bookReducer = (state = [], action) => {
+  switch (action.type) {
+    case 'ADD':
```

Comment:
- [ ] It is better to use the variable action Type here instead of using a hardcoded one, that is why you need to have defined your action types somewhere and just call them here.